recipes/libredwg/build.sh

```diff
@@ -0,0 +1,27 @@
+echo "####################################"
+echo $(ls -al)
+echo "####################################"
+
+# TODO: Fix this upstream
+mv git-version-gen build-aux/git-version-gen
```

Comment:
Can you open an issue upstream to get this included in the release tarballs? I'm not sure why it's only in the repository.

Reply:
@chrisburr thanks for having a look at this. I guess it's best to ping @rurban here.